test/integrationTests/testAssets/spawnGit.ts

```diff
@@ -25,8 +25,8 @@ export default async function spawnGit(args?: string[], options?: SpawnOptions):
     
     let spawned = spawn('git', args, optionsWithFullEnvironment);
 
-    spawned.stdout.on('data', (data) => console.log(data.toString()));
-    spawned.stderr.on('data', (data) => console.log(data.toString()));
+    //spawned.stdout.on('data', (data) => console.log(data.toString()));
+    //spawned.stderr.on('data', (data) => console.log(data.toString()));
```

Comment:
Did you mean to leave this commented or should it just be deleted?

Reply:
I have left if commented. It might be useful to enable it if sometimes we need to debug.